instrumentation/http.js

```diff
@@ -108,12 +108,26 @@ function instrumentHTTP(mod) {
             'request.scheme': u.protocol.slice(0, -1),
           });
           req.once('response', res => {
-            span.addContext({
-              'response.http_version': res.httpVersion,
-              'response.status_code': res.statusCode,
-              ...responseHeaderFields(res.headers),
+            let responseBody = '';
+            res.on('data', chunk => {
```

Comment:
Presumably this doesn't affect the app consuming the data too? Eg in go we have to do be very careful to buffer the body then return a reader or the in memory body - I'm assuming we don't have to do that here?

Reply:
Couldn't seem to find a definitive answer in the docs, but I've added an assertion in the test and it works 👍